@troopa81 It looks like the clang-tidy CI does not work at the moment. One get the following error on each inspected file: 

```
error: PCH file uses an older PCH format that is no longer supported [clang-diagnostic-error]
```

See for example: 
- https://github.com/qgis/QGIS/actions/runs/13362091067/job/37314779975?pr=60627
- https://github.com/qgis/QGIS/actions/runs/13340766744/job/37265791702?pr=60605
- https://github.com/qgis/QGIS/actions/runs/13374511384/job/37355486087?pr=60646

This has nothing to do with the change introduced here.

--- a/.clang-tidy
+++ b/.clang-tidy
@@ -1,1 +1,2 @@
 Checks: 'bugprone-*,-bugprone-easily-swappable-parameters,-bugprone-virtual-near-miss,-bugprone-suspicious-include'
+HeaderFilterRegex: ''

--- a/external/.clang-tidy
+++ b/external/.clang-tidy
@@ -1,0 +1,4 @@
+# clang-tidy does not allow easy deactivation of a directory.
+# Use a dummy .clang-tidy file that disables all clang-tidy checks except one that will never match.
+# This one check is necessary; clang-tidy reports an error when no checks are enabled.
+Checks: '-*,llvm-twine-local'

